webapp/channels/src/components/suggestion/command_provider/app_provider.tsx

```diff
@@ -31,19 +31,11 @@ type Props = {
     rootId?: string;
 };
 
-export type Results = {
-    matchedPretext: string;
-    terms: string[];
-    items: Array<AutocompleteSuggestion | UserProfile | {channel: Channel}>;
-    component?: React.ElementType;
-    components?: React.ElementType[];
-}
-
-type ResultsCallback = (results: Results) => void;
+type Item = AutocompleteSuggestion | UserProfile | {channel: Channel};
 
 export default class AppCommandProvider extends Provider {
     private store: Store<GlobalState>;
-    private triggerCharacter: string;
+    public triggerCharacter: string;
```

Comment:
This needs to be public because it's used by the post textbox to position its autocomplete popup differently for different types of autocompletes